test/client-checkout-integration-test.js

```diff
@@ -16,6 +16,7 @@ import checkoutUpdateAttributesV2Fixture from '../fixtures/checkout-update-custo
 import checkoutUpdateEmailV2Fixture from '../fixtures/checkout-update-email-fixture';
 import checkoutDiscountCodeApplyV2Fixture from '../fixtures/checkout-discount-code-apply-fixture';
 import checkoutDiscountCodeRemoveFixture from '../fixtures/checkout-discount-code-remove-fixture';
+import checkoutShippingAddressUpdateFixture from '../fixtures/checkout-shipping-address-update-fixture';
```

Comment:
The fixture should have `V2` in the name: `checkoutShippingAddressUpdateV2Fixture`